src/Styles/StyledComponents.js

```diff
@@ -184,6 +198,12 @@ export const Container = styled.div`
   margin-right: auto;
 `;
 
+export const SmallContainer = styled.div`
```

Comment:
what does this contain? Should it be named accordingly? `CodeOfConductContainer` for example